packages/core/rpc/WebWorkerRpcDriver.ts

```diff
@@ -46,21 +41,23 @@ class WebWorkerHandle extends Rpc.Client {
 export default class WebWorkerRpcDriver extends BaseRpcDriver {
   name = 'WebWorkerRpcDriver'
 
-  WorkerClass: WebpackWorker
+  makeWorkerInstance: () => Worker
```

Comment:
rootModel passes a makeWorkerInstance callback which calls new Worker now https://webpack.js.org/guides/web-workers/ and tests mock this out to return, since jest doesn't really like the import.meta.url  and we use mainthreadrpc anyways